CONTRIBUTING.md

```diff
@@ -67,8 +67,12 @@ public extension SomeType {
 	- Properties & static properties
 	- Methods & static methods
 	- Initializers
-- Files are named based on the type that the contained extensions extend (example: all String extensions are found in "**StringExtensions.swift**" file)
-- extensions are ordered inside files in the following order:
+- Files are named based on the type that the contained extensions extend. (example: all String extensions are found in "**StringExtensions.swift**" file)
+- All extensions should be tested.
+- All extensions files and test files have a one to one relation.(example: all tests for "**StringExtensions.swift**" are found in the "**StringExtensionsTests.swift**" file)
+- All test files are named based on the extensions which it tests. (example: all String extensions tests are found in "**StringExtensionsTests.swift**" file)
+- An single extension can have multiple tests.
```

Comment:
I think here we have to explain the one to one extension-test methods relation. Example: the extension Array.rotate() should have one testRotate() test on ArrayTests file where all the edge cases are for this are tested inside this test :)